main.go

```diff
@@ -71,10 +71,10 @@ func main() {
 
 	server := &http.Server{
 		Addr:              ":8000",
-		ReadTimeout:       5 * time.Second,
-		WriteTimeout:      5 * time.Second,
+		ReadTimeout:       30 * time.Second,
+		WriteTimeout:      30 * time.Second,
 		IdleTimeout:       30 * time.Second,
-		ReadHeaderTimeout: 3 * time.Second,
+		ReadHeaderTimeout: 30 * time.Second,
```

Comment:
I'm curious how do you determine the timeout threshold, by test deploying to the production server?